packages/adblocker/test/resources.test.ts

```diff
@@ -24,70 +24,97 @@ describe('#Resources', () => {
     expect(Resources.deserialize(buffer)).to.eql(resources);
   });
 
-  describe('#parse', () => {
-    it('parses empty resources', () => {
-      const resources = Resources.parse('', { checksum: 'checksum' });
-      expect(resources.checksum).to.equal('checksum');
-      expect(resources.js).to.eql(new Map());
-      expect(resources.resources).to.eql(new Map());
-    });
+  context('#parse', () => {
+    it('parses dependencies', () => {
+      const distribution: ResourcesDistribution = {
+        redirects: [],
+        scriptlets: [
+          {
+            names: ['a'],
+            content: 'function a() { b() }',
+            dependencies: ['b'],
+          },
+          {
+            names: ['b'],
+            content: 'function b() {}',
+            dependencies: [],
+          },
+        ],
+      };
+      const resources = Resources.parse(JSON.stringify(distribution), { checksum: '' });
 
-    it('parses one resource', () => {
-      const resources = Resources.parse('foo application/javascript\ncontent', {
-        checksum: 'checksum',
-      });
-      expect(resources.checksum).to.equal('checksum');
-      expect(resources.js).to.eql(new Map([['foo', 'content']]));
-      expect(resources.resources).to.eql(
-        new Map([['foo', { contentType: 'application/javascript', body: 'content' }]]),
+      expect(resources.getScriptlet('a')).to.be.eql(
+        wrapScriptletBody('function a() { b() }', ['function b() {}']),
       );
+      expect(resources.getScriptlet('b')).to.be.eql(wrapScriptletBody('function b() {}', []));
```

Comment:
Not sure if we should use `wrapScriptletBody` (which btw deserves own tests). Can we test if `resources.scriptlets` are populated correctly?